ToDoListSrc/src/todolist/modelos/Tarea.java

```diff
@@ -5,50 +5,18 @@
  */
 package todolist.modelos;
 
-import java.util.Date;
-
 /**
  *
  * @author tecnologer
  */
 public class Tarea {
-    private int IDUsuario;
     private String descripcion;
     private boolean estaCompletada;
-    private Date fechaInicio;
```

Comment:
Hay que usar fechas con tiempo y zona horaria, el tipo recomendado aqui seria [`OffsetDateTime`](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html#:~:text=OffsetDateTime%20is%20an%20immutable%20representation,be%20stored%20in%20an%20OffsetDateTime%20.)